bin/sngl/pycbc_export_censored_trigs

```diff
@@ -102,8 +105,39 @@ else:
     trigs_censored = ForegroundTriggers(opts.coinc_file, opts.bank_file,
                           sngl_files=opts.sngl_files, group=group)
 
-# find triggers above limit
-mask = numpy.where( trigs_censored.get_coincfile_array("stat") > opts.min_stat )[0]
+# apply clustering to censored_triggers if a cluster-window has been parsed in the command line else skip clustering. Then pick triggers above min-stat if one is parsed in the command line
+
+# The clustering picks the loudest trigger in each window
+
+if opts.cluster_window:
+    coinc_newsnr = numpy.array( trigs_censored.get_coincfile_array("stat") )
```

Comment:
It is necessary to keep typecasting everything ``numpy.array``? There are several in this code block and in all cases these should already be ``numpy.array``.